### PR TITLE
Raven db 9719

### PR DESCRIPTION
**Pull request by aviv86**

### CODE REVIEW

File: src/Raven.Client/Documents/Changes/ChangeNotification.cs [hunk 1]
```diff
@@ -174,6 +174,6 @@ public class TrafficWatchChange : DatabaseChange
         public string DatabaseName { get; set; }
         public string CustomInfo { get; set; }
         public int InnerRequestsCount { get; set; }
-        public object QueryTimings { get; set; } // TODO: fix this
+        public object QueryTimings { get; set; }
```
Comment (ayende) [inline src/Raven.Client/Documents/Changes/ChangeNotification.cs:177]: I think this needs an issue, it should be hidden behind an `#if`, IIRC

File: src/Raven.Client/ServerWide/DatabaseRecord.cs [hunk 2]
```diff
@@ -46,7 +46,6 @@ public DatabaseRecord(string databaseName)
 
         public Dictionary<string, string> Settings = new Dictionary<string, string>();
 
-        //todo: see how we can protect this
         public Dictionary<string, string> SecuredSettings;
```
Comment (ayende) [inline src/Raven.Client/ServerWide/DatabaseRecord.cs:49]: IIRC, we can _remove_ this entirely.

File: src/Raven.Server/Commercial/LicenseManager.cs [hunk 3]
```diff
@@ -1071,8 +1071,6 @@ private void ThrowIfCannotActivateLicense(LicenseStatus newLicenseStatus)
             {
                 foreach (var databaseRecord in _serverStore.Cluster.ReadAllDatabases(context))
                 {
-                    //TODO: check monitoring existance
```
Comment (ayende) [inline src/Raven.Server/Commercial/LicenseManager.cs:1074]: Shouldn't we check monitoring here?

File: src/Raven.Server/Documents/Handlers/StreamingHandler.cs [hunk 4]
```diff
@@ -21,7 +21,6 @@ public class StreamingHandler : DatabaseRequestHandler
         public Task StreamDocsHead()
```
Comment (ayende) [inline src/Raven.Server/Documents/Handlers/StreamingHandler.cs:21]: Pretty sure that we can delete this entirely.

File: src/Raven.Client/Documents/Changes/ChangeNotification.cs (hunk 1 above)
Comment (aviv86) [reply src/Raven.Client/Documents/Changes/ChangeNotification.cs:177]: creating an issue for that

File: src/Raven.Server/Commercial/LicenseManager.cs (hunk 3 above)
Comment (aviv86) [reply src/Raven.Server/Commercial/LicenseManager.cs:1074]: yes, creating an issue for that

**Pull request merged**